FinderCompass/src/main/java/atomicstryker/findercompass/common/DefaultConfigFilePrinter.java

```diff
@@ -162,22 +170,27 @@ else if (settingList.isEmpty())
                             }
                             else
                             {
-                                configInts[8] = -1;
+                                configInts[8] = 0;
                             }

```

Comment:
I've now read the description stating this. Uh so what about users who have -1 in their configs?